cedar-policy-core/src/evaluator.rs

```diff
@@ -235,7 +234,11 @@ impl<'e> Evaluator<'e> {
     ///    it doesn't consider whether we're processing a `Permit` policy or a
     ///    `Forbid` policy.
     pub fn partial_evaluate(&self, p: &Policy) -> Result<Either<bool, Expr>> {
-        match self.partial_interpret(&p.condition(), p.env())? {
+        self.partial_evaluate_with_mappings(p, None)
+    }
+
+    pub(crate) fn partial_evaluate_with_mappings(&self, p: &Policy, mappings: Option<&HashMap<SmolStr, Value>>) -> Result<Either<bool, Expr>> {
```

Comment:
Doc comment here (describing that `mappings` is for unknowns, and not, say, template slots) would be nice